build/Program.fs

```diff
@@ -170,9 +260,11 @@ let clear() =
     File.Delete(path [ solutionRoot; "src"; "Spotify.fsproj" ])
     File.Delete(path [ solutionRoot; "src"; "Spotify.Dotnet.fsproj" ])
     File.Delete(path [ solutionRoot; "src"; "Spotify.Fable.fsproj" ])
+    File.Delete(path [ solutionRoot; "src"; "SpotifyWithTasks.fsproj" ])
     Directory.Delete(path [ solutionRoot; "src"; "output" ], true)
 
 let integration() =
+    //tasksIntegration()
```

Comment:
What is this for exactly and why is it commented out?

Reply:
I wrote integration tests for MSBuild tasks. Accidentally added these changes to the commit.